psi4/src/psi4/psimrcc/mp2_ccsd.cc

```diff
@@ -152,10 +152,14 @@ void MP2_CCSD::compute_mp2_ccsd_energy() {
     outfile->Printf("\n\n   * MP2-CCSD total energy = %25.15f\n", current_energy);
 
     compute_mp2_ccsd_components();
+
+    ref_wfn_->set_scalar_variable("CURRENT ENERGY", current_energy);
+    ref_wfn_->set_scalar_variable("MP2-CCSD TOTAL ENERGY", current_energy);
+    return ref_wfn_->scalar_variable("CURRENT ENERGY");
```

Comment:
For anyone worried about this using qcvars instead of member data, be assured that the `CURRENT`s have been tied into the fundamental e/g/h_ quantities: https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libmints/wavefunction.cc#L1262-L1273

Reply:
Except for where `perturb_cbs` is concerned, but I'll hopefully be fixing that.

Reply:
You definitely mean `ref_wfn_` here, not the wfn itself? Usually the reference wfn wouldn't have CC quality. I can believe psimrcc is the exception, so just checking.

Reply:
`psimrcc` doesn't have a wavefunction yet! The next PR should give it one, and all genuinely `psimrcc` quantities (like this one) will then go on `psimrcc`'s wavefunction.